app/utility/base_world.py

```diff
@@ -122,7 +122,7 @@ def check_module_version(module, version, attr=None, **kwargs):
             return compare_versions(mod_version, version)
 
         def check_program_version(command, version, **kwargs):
-            output = subprocess.check_output(command.split(' '), shell=False)
+            output = subprocess.check_output(command.split(' '), stderr=subprocess.STDOUT, shell=False)
```

Comment:
I just want to check, what's the effect of this change?  Does this mean stderr is redirected to stdout, which in turn gets saved into the output var (e.g. both stderr and stdout are mingled in output?)?

Reply:
Yeah, both are mingled in the subprocess output. Without this change metasploit's version would be parsed as `''` because metasploit seems to ouput its version info to stderr.